types/output.d.ts

```diff
@@ -7,6 +7,7 @@ import { EventHubOutput, EventHubOutputOptions } from './eventHub';
 import { GenericOutputOptions } from './generic';
 import { HttpOutput, HttpOutputOptions } from './http';
 import { FunctionOutput } from './index';
+import { MySqlOutput, MySqlOutputOptions } from './mySql';
```

Comment:
Looks more like an import reorder - was this triggered during some linting? Is there a code style file that is included?